src/config.rs

```diff
@@ -15,6 +15,10 @@ lazy_static::lazy_static! {
         RwLock::new(HashMap::new());
 }
 
+pub fn config_file_name() -> &'static str {
+    CONFIG_FILE_NAME
+}
+
```

Comment:
why this utility function? Can `CONFIG_FILE_NAME` be made public and used directly in `handler.rs`?